src/components/routes.js

```diff
@@ -1,5 +1,5 @@
-import DeveloperPreview from './developer-preview';
-import Introduction from './introduction';
+// import DeveloperPreview from './developer-preview';
+import Introduction from './introduction-v2';
```

Comment:
Rather than adding a v2 can you edit the existing introduction? We can always go back and revert if there are parts we want to keep. Same goes for deleting the developer preview page.